src/app/user-login/login/login.component.ts

```
@@ -151,11 +155,15 @@
             if (response.data) {
               if (response.data.previlegeObj.length === 0) {
                 console.log(response.data, 'SUPERADMIN VALIDATED');
+                console.error('Jwttoken', response);
+                console.log('Jwttoken', response);
+                this.cookieService.set('Jwttoken', response.data.Jwttoken);
+                delete response.data.Jwttoken;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)